### ISSUE TITLE
Conflict. The container name "/c3" is already in use..

### ISSUE THREAD

**Issue by gabrielgiussi** [RBMHTechnology/eventuate-chaos#15]
Hi, I'm getting this error when I run blockade up with the example app.
I'm using docker 1.13.0
![blockade-error](https://cloud.githubusercontent.com/assets/7468185/22190129/db8b4dc8-e0ff-11e6-96fb-72fe721f0229.png)

**Comment by volkerstampa** [comment]
Hi @gabrielgiussi ,
to me this looks like a left over from a previous run that maybe failed prematurely. Could you stop running containers and remove them before a retry? With your example listed above the following should do: 

    docker kill c1
    docker rm c1 c3

In addition to this the an entry in your `resolv.conf` for dnsdock is required. Best is to start `./start-dns.sh -f` for this. After that you could retry with:

    blockade up

**Comment by kongo2002** [comment]
Hi @gabrielgiussi ,
I think @volkerstampa is right here - by default `blockade` refuses to start a container if one with the same name already exists. However you may force that by invoking `blockade up -f` instead.

**Comment by gabrielgiussi** [comment]
You were right, I thought I had checked that, my bad. Also, the ./start-dns.sh -f worked well ) although I don't understood what it does)
However now I get a different kind of error and I am not familiar with Python.

![blockade](https://cloud.githubusercontent.com/assets/7468185/22274897/ccd2dea2-e287-11e6-8297-7bf5f1ef15dd.png)

The containers appear to have started well but both blockade up as blockade status threw that error.
Python version 2.7.9

A sidenote, I'm new to docker but as far as I understand the image of blockade is kind of heavy (1.29 GB) and the java8 image has been [deprectaed](https://hub.docker.com/_/java/). Maybe we could use 8-jdk- alpine? Just a comment.

Thanks for the help.

**Comment by volkerstampa** [comment]
- `start-dns.sh -f` adds the dns service that is used by the docker containers to your `resolv.conf` so the docker containers know about it and can find themselves by name
- the python exception tries to tell you that blockade cannot find the `tc` command. From your prompts I would assume that you do not run blockade as root, so please retry with `sudo blockade up`
- Thanks for the hint with the jdk-image. Could you please create an issue for this. It probably makes more sense to have a look into this once #14 is merged as the `Dockerfile` is much simpler then.

**Comment by gabrielgiussi** [comment]
@volkerstampa It was that indeed, I've executed `sudo blocakde up` and everything worked. I have to say that it wasn't my brightness day, sorry and thanks to both for the help. I'll close the issue.
I've created the issue as you asked, I hope it's clear.

**Comment by volkerstampa** [comment]
@gabrielgiussi Good to hear that it works. Thanks for opening the new issue. It is prefectly clear.
